app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RelationshipsController < ApplicationController
+  before_action :require_user
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def create
+    leader = User.find(params[:leader_id])    
+    Relationship.create(leader_id: params[:leader_id], follower_id: current_user.id) if current_user.can_follow?(leader)
+   
+    redirect_to people_path
+  end
+
+  def destroy
+    @relationship = Relationship.find(params[:id])
+    @relationship.delete if @relationship.follower_id == current_user.id
```

Comment:
Try `current_user.relationship.find(params[:id]).delete` instead of these two lines